contrib/hbcurl/core.c

```diff
@@ -1034,6 +1051,50 @@ HB_FUNC( CURL_EASY_SETOPT )
                }
             }
             break;
+            case HB_CURLOPT_HTTPPOST_FORM:
+            {
+               PHB_ITEM pArray = hb_param( 3, HB_IT_ARRAY );
+
+               if( pArray )
+               {
+                  HB_SIZE ulPos;
+                  HB_SIZE ulArrayLen = hb_arrayLen( pArray );
+
+                  for( ulPos = 0; ulPos < ulArrayLen; ++ulPos )
+                  {
+                     PHB_ITEM pSubArray = hb_arrayGetItemPtr( pArray, ulPos + 1 );
+
+                     int type = hb_arrayGetNI( pSubArray, 1 );
+
+                     switch( type )
+                     {
+                        case 1:
```

Comment:
Instead of introducing arbitrary numeric constant here, it'd be best to drop it, 
keep case `1` under the name `HB_CURLOPT_HTTPPOST_CONTENT`.
This is already implemented in 3.4 fork.